test/ReactWrapper-spec.jsx

```diff
@@ -1807,6 +1807,12 @@ describeWithDOM('mount', () => {
       expect(children.at(1).hasClass('baz')).to.equal(true);
     });
 
+    it('should not attempt to get an instance for text nodes', () => {
+      const wrapper = mount(<div>B<span />C</div>);
+      const children = wrapper.children();
+      expect(children.length).to.equal(1);
```

Comment:
That's inconsistent though. Before this change, mounting an element with a single text node child returns `0` children.

```js
      const wrapper = mount(<div>A</div>);
      wrapper.children().length
      // 0
```